Library/Formula/premake5.rb

```diff
@@ -0,0 +1,18 @@
+class Premake5 < Formula
```

Comment:
Basically unless this breaks Homebrew-stuff using `premake`: we should package the latest version.